src/client/app/components/AdminComponent.jsx

```diff
@@ -3,15 +3,49 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
+import Dropzone from 'react-dropzone';
+import axios from 'axios';
+import MeterDropDownContainer from '../containers/MeterDropdownContainer';
+import HeaderContainer from '../containers/HeaderContainer';
 
-/**
- * React component that controls the Admin panel
- * @returns JSX to create the AdminComponent
- */
-export default function AdminComponent() {
-	return (
-		<div>
-			<p>Admin panel</p>
-		</div>
-	);
+export default class AdminComponent extends React.Component {
+	constructor(props) {
+		super(props);
+		this.handleOnDrop = this.handleOnDrop.bind(this);
+	}
+
+	handleOnDrop(files) {
```

Comment:
Maybe specify that we are handling meter import. On Drop could refer to anything in the admin panel.